DEVELOPMENT.md

```diff
@@ -14,12 +14,16 @@ kubectl api-resources
 # Build and run Cluster Controller
```

Comment:
actually I think the whole section should go. cluster-controller is still there, but totally untested and kcp itself has the same controller integrated. So nobody has to run it in its own process. 

Also installing the CRDs via `kubectl apply` is not needed anymore.

Reply:
Finally `# Test the registration of a Physical Cluster` is also a relict of the past. Push mode is gone in main branch, and there is a cli plugin (`kubectl workloads sync`) to setup a syncer.